types/rc-tooltip/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for rc-tooltip v3.4.2
+// Type definitions for rc-tooltip v3.7.3
```

Comment:
Two requests:
- Please make the version string `3.7` (no patch version or `v`)
- Please delete the line in the `tslint.json` that says `dtheader: false` so that the header can be validated.

It would be great to remove every rule disabling line in the `tslint.json`, but that might be a lot of work.

Reply:
Done!